core/api/build.gradle.kts

```diff
@@ -64,3 +72,11 @@ dependencies {
     add("kspIosArm64", libs.ktorfit.ksp)
     add("kspIosSimulatorArm64", libs.ktorfit.ksp)
 }
+
+kotlin.sourceSets.commonMain {
+    kotlin.srcDir("build/generated/ksp/metadata/commonMain/kotlin")
+}
+
+tasks.withType<KotlinCompile> {
+    dependsOn(tasks.withType<KspTaskMetadata>())
+}
```

Comment:
These are needed:
- the first block makes sure the code in `build/generated/ksp/metadata` is visible in the commonMain source set
- the second one ensures code generation is run, otherwise when building the module the KSP task is skipped.